.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,151 @@
+          required: false
+        - label: If you are interested in working on this issue or have submitted a pull request, please leave a comment.
+          required: false
+  - type: input
+    id: version-terraform
+    attributes:
+      label: Terraform
+      description: Please provide the Terraform version.
+      placeholder: e.g. x.y.z
+    validations:
+      required: true
+  - type: input
+    id: version-provider
+    attributes:
+      label: Terraform Provider
+      description: |
+        Please provide the provider version.
+        We recommend testing with [the latest version of the provider](https://github.com/vmware/terraform-provider-nsxt/releases/latest).
+      placeholder: e.g. x.y.x
+    validations:
+      required: true
+  - type: input
+    id: version-nsx
+    attributes:
+      label: VMware NSX
+      description: |
+        Please provide the VMware NSX version.
+      placeholder: e.g. x.y.z
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Please provide a clear and concise description of the issue you are experiencing.
+    validations:
+      required: true
+  - type: textarea
+    id: affected-resources-datasources
+    attributes:
+      label: Affected Resources or Data Sources
+      description: Please provide the affected resources or data sources as a list.
+      placeholder: |
+        e.g. `resource/example` or `datasource/example`
+    validations:
+      required: true
+  - type: textarea
+    id: configuration
+    attributes:
+      label: Terraform Configuration
+      description: |
+        - Please provide a code repository, [GitHub Gist](https://gist.github.com/), or simplified code sample suitable for issue reproduction.
+        - Please ensure all secrets and identifiable information is removed.
+        - Please format any code in code blocks using the markdown editor.
+
+        Learn more about [Code Formatting](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).
+      placeholder: |
+        Please format any code in code blocks using the markdown editor.
+    validations:
+      required: true
+  - type: textarea
+    id: debug
+    attributes:
+      label: Debug Output
+      description: |
+        Please provide a link to a [GitHub Gist](https://gist.github.com/) containing the complete debug output.
+      placeholder: |
+        Link to a GitHub Gist. Please do not paste the debug output in the issue.
+    validations:
+      required: true
+  - type: textarea
+    id: panic
+    attributes:
+      label: Panic Output
+      description: |
+        If Terraform produced a panic, please provide a link to a [GitHub Gist](https://gist.github.com/) containing the output of the `crash.log`
+      placeholder: |
+        Link to a GitHub Gist. Please do not paste the panic output in the issue.
```

Comment:
As above. Often the panic is easily identified by a small stack trace.

Reply:
Resolved in https://github.com/vmware/terraform-provider-nsxt/commit/50fcb7ad1539140b4f6f4a14864b57ee36196e59.